src/main.rs

```diff
@@ -88,8 +88,8 @@ fn run_command(log: slog::Logger, opts: Arguments) -> OpResult {
 
     match opts.command {
         Command::Info(opts) => {
-            let (project, _guard) = with_project(&opts.nix_file)?;
-            info::main(project)
+            let (_project, _guard) = with_project(&opts.nix_file)?;
```

Comment:
Not using any of the fields, so we can remove this line.

Reply:
The `_guard` is required. I believe that calling things `_guard` is something of a convention which indicates that the variable, though unused, must stay in scope.

In this case, the `_guard` protects the thread-local logger from being destroyed.

`_project` is indeed unused.

Would you prefer the following?

```rust
let _guard = with_project(&opts.nix_file)?.1;
```

Reply:
Oh, right, we need it for scoping. Then it’s fine. I didn’t know about the `_guard` convention.